### PR TITLE
(DO NOT MERGED YET) Prettier for ESLint

### PR DESCRIPTION
**Pull request by mmmavis**
# DO NOT MERGE YET

Related issue: #1052

~~There are 1165 formatting issues found so far 😮  so I think it's best that we keep this PR open until work in this repo slows down so we can fix all the linting errors and merge this PR to avoid crazy merge conflicts. Thoughts @alanmoo ? If you think we should get this landed sooner I guess I can temporarily make `prettier` raise warnings instead of errors so these to-be-fixed issues don't break our test and we can fix linting issues bits by bits.~~

Going with the `warnings` approach

### CODE REVIEW

**Comment by alanmoo** [comment]
Go for the warnings approach, that sounds reasonable.

**Comment by mmmavis** [comment]
Fixed linting errors in some files that are not touched by any open PRs at the moment.

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (mmmavis) [inline .eslintrc.json]: copied from https://github.com/mozilla/foundation.mozilla.org/blob/master/.eslintrc.json

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "sourceType": "module",
+      "ecmaFeatures": {
+        "jsx": true
+      }
+    },
+    "env": {
+      "es6": true
+    },
+    "rules": {
+      "prettier/prettier": "warn",
```
Comment (mmmavis) [inline .eslintrc.json]: set to `warn` for now

**Comment by alanmoo** [comment]
Is this ready for review now?

**Comment by mmmavis** [comment]
@alanmoo yup! We can address other linting errors as follow-up PRs. I'm worried if I go too far I will mess up Youri's work.

**Comment by mmmavis** [comment]
Oops! Forgot to remove `(DO NOT MERGED YET)` from the title before I merge this.

**Comment by mmmavis** [comment]
I can fix the commit message if we want to: https://github.com/mozilla/network-pulse/commits/master 😂

**Comment by alanmoo** [comment]
Hah. Seems risky. I don’t think it’s an issue.

**Pull request merged**